compiler/rustc_feature/src/active.rs

```diff
@@ -516,6 +516,8 @@ declare_features! (
     /// Allows dyn upcasting trait objects via supertraits.
     /// Dyn upcasting is casting, e.g., `dyn Foo -> dyn Bar` where `Foo: Bar`.
     (active, trait_upcasting, "1.56.0", Some(65991), None),
+    /// Allows for transmuting between arrays with sizes that contain generic consts.
+    (active, transmute_generic_consts, "1.7.0", Some(109929), None),
```

Comment:
Maybe 1.70.0?

Reply:
Shouldn't this be CURRENT_RUSTC_VERSION?

Reply:
maybe

Reply:
Should be `CURRENT_RUSTC_VERSION` instead of explicit version.